imagefiles/install-crosstool-ng-toolchain.sh

```diff
@@ -111,6 +111,9 @@ cp "${CONFIG_PATH}" "${BUILD}/.config"
 
 # As mentioned in ct-ng config, need to unset LD_LIBRARY_PATH.
 unset LD_LIBRARY_PATH
+# Fix build error on manylinux2014-aarch64
+unset CC
+unset CXX
```

Comment:
Just to be sure: does it still build projects after those two are unset? I don't use manylinux2014-aarch64 myself so I wouldn't know :grin:

Reply:
> Just to be sure: does it still build projects after those two are unset? I don't use manylinux2014-aarch64 myself so I wouldn't know

On newer versions of crosstool-ng, it gives me an error if I don't unset the CC and CXX variables,
I failed to update crosstool-ng on manylinux 2014-aarch64,
this PR can be useful if one day a person updates crosstool-ng to a newer version, but if we stay on version 1.23, this PR is useless

Reply:
But aren't `CC` and `CXX` used e.g. by cmake? What happens when they are unset?

Reply:
> But aren't `CC` and `CXX` used e.g. by cmake? What happens when they are unset?

I think crosstool-ng uses these values, when they are initialized with values, crosstool-ng gives me an error during the build.
later they are initialized again with values :
https://github.com/bensuperpc/dockcross/blob/39a31c83b1a1457815678c21d29c8e0d31d69691/manylinux2014-aarch64/Dockerfile.in#L29

This is not a problem unless a script or software uses the value without having a default value if it is empty,
This does not appear to be the case, because the tests are working.

On Manjaro, they are not initialized, CMake works well

```
[bensuperpc@bensuperpc-luna dockcross]$ echo "C compiler:$CC"
C compiler:
[bensuperpc@bensuperpc-luna dockcross]$ echo "C++ compiler:$CXX"
C++ compiler:
```
I think CMake is good enough to automatically find default compilers